python/tchannel/tornado/util.py

```diff
@@ -52,3 +53,21 @@ def print_arg(request, index):
                 print chunk
     except Exception:
         pass
+
+
+def deserialize_default(obj):
+    return obj
+
+
+def serialize_default(obj):
+    return obj
+
+
+def deserialize_json(obj):
+    assert obj
+    return json.loads(obj)
+
+
+def serialize_json(obj):
+    assert obj
+    return json.dumps(obj)
```

Comment:
Actually, `json.dumps(None)` is the string `"null"`.

Only deserialize doesn't know how to handle empty strings or None.